src/Helpers/Builder/Variable.php

```diff
@@ -47,7 +47,7 @@ public function setSubstitutions(array $substitutions): void
         foreach ($substitutions as $var => $value) {
             $mapped[] = [
                 'var' => $var,
-                'value' => $value,
+                'value' => (string) $value,
```

Comment:
This would not work as f.e a boolean could be interpreted wrong. Need to add a more broader stringify helper if we are going to force such a thing.

Reply:
so what do you expect when a boolean converts to a string? (string) converts it to '1' - do you expect a 'true' or 'false' in such cases?